frontend/packages/dev-console/integration-tests/features/addFlow/developer-Catalog-Details.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Developer Catalog Page
+    As a user, I should be able to use Developer Catalog page to deploy application
+
+
+    Background:
+        Given user is at developer perspective
+        And user has installed OpenShift Serverless Operator
+        And user has installed Knative Apache Camelk Operator
+        And user has installed Apache Kafka Operator
+    
+
+    @smoke
+    Scenario: Developer Catalog page
+        Given user is at Add page
+        When user clicks on From Catalog card
+        Then user will see All Items already selected
+        And user will see CI/CD, Databases, Languages, Middleware, Other categories
+        And user will see Builder Images, Event Sources, Helm Charts, Operator Backed, Templates types
+        And user will see Filter by Keyword field
+        And user will A-Z, Z-A sort by dropdown
+
+    
+    @smoke, @manual
+    Scenario: Event Sources on Developer Catalog
+        Given user is at Developer Catalog page
+        When user clicks on Event Sources type
+        Then user will see the list of Providers
+        And user will see the cards of Event Sources
+        And user will see Filter by Keyword field
+        And user will A-Z, Z-A sort by dropdown
+
+
+    @smoke, @manual
+    Scenario: Helm Charts on Developer Catalog
+        Given user is at Developer Catalog page
+        When user clicks on Helm Charts type
+        Then user will see the list of Chart Repositories
```

Comment:
Same as above. Not always true.

Reply:
Updated

Reply:
How is this resolved? It still says the same thing.